client/src/common/constants.ts

```diff
@@ -1 +1,4 @@
 export const EMPTY_ARRAY = [];
+
+// Column name for gene names in the annomatrix
```

Comment:
Not a blocker for now, but my only concern with this approach is that it's not as flexible as the original design's intention to allow the dataset to decide which column to use as feature anymore?

If we're seeing issues with it, would it be possible that it's the datasets that's passing the wrong feature column as index?